src/index.html

```diff
@@ -17,16 +17,27 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-bar"
+      data-qa="big"
+    >
```

Comment:
The `<form>` tag should not be self-closed. Forms in HTML should have an opening and a closing tag. Please change `<form />` to `<form>` and ensure it has a corresponding `</form>` closing tag.